akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -893,7 +893,17 @@ object Unzip {
 }
 
 /**
- * Combine the elements of multiple streams into a stream of the combined elements.
+ * Takes a stream of pair elements and splits each pair to two output streams.
+ *
+ * An `Unzip` has one `in` port and one `left` and one `right` output port.
+ *
+ * '''Emits when''' all of the outputs stops backpressuring and there is an input element available
+ *
+ * '''Backpressures when''' any of the outputs backpressures
```

Comment:
I think 'any of the outputs backpressures' should be 'any of the outputs backpressure' here?